config/protocol.unit_testnet.single.yml

```diff
@@ -20,6 +20,8 @@ ProtocolConfiguration:
     RoleManagement: [0]
     OracleContract: [0]
     Notary: [0]
+  Hardforks:
```

Comment:
Hmm, but we have not documented it! And it's not an easy one.

Reply:
Not sure, should we explain why hardforks have been added?

Reply:
The concept itself is pretty well-known, but I'll take at look at the text.